### PR TITLE
CI: praktika integration 1

### PR DESCRIPTION
**Pull request by maxknv**
1. adds (not-enabled) workflow written with praktika
2. atm workflow includes the only one configured docker - style-test, written in praktika's style
3. atm workflow includes the only one job - style check, cleaned from ci infrastructure code, docker and wrappers

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-2** [comment]
*This is an automated comment for commit 738588f692efe910a880ca15c80c9d6501d07615 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/69822/738588f692efe910a880ca15c80c9d6501d07615/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://clickhouse-builds.s3.amazonaws.com/artifacts/reports/json.html?results=https%3A%2F%2Fclickhouse-builds.s3.amazonaws.com%2Fartifacts%2Freports%2F69822%2Flatest%2Fresult_pr.json">⏳ pending</a></td></tr>
<tbody>
</table>
<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/924e21f632d22ad26b8380f3286666715d448270/docs_check.html">✅ success</a></td></tr>
<tr><td>Fast test</td><td>Normally this is the first check that is ran for a PR. It builds ClickHouse and runs most of <a href="https://clickhouse.com/docs/en/development/tests#functional-tests">stateless functional tests</a>, omitting some. If it fails, further checks are not started until it is fixed. Look at the report to see which tests fail, then reproduce the failure locally as described <a href="https://clickhouse.com/docs/en/development/tests#functional-test-locally">here</a></td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/aa5e76ae8785387212baca6e79a761abf36cf65c/fast_test.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/aa5e76ae8785387212baca6e79a761abf36cf65c/unit_tests__release_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: utils/check-style/check-style [hunk 1]
```diff
@@ -285,9 +285,6 @@ done
 # There shouldn't be any code snippets under GPL or LGPL
 find $ROOT_PATH/{src,base,programs} -name '*.h' -or -name '*.cpp' 2>/dev/null | xargs grep -i -F 'General Public License' && echo "There shouldn't be any code snippets under GPL or LGPL"
 
-# There shouldn't be any docker containers outside docker directory
-find $ROOT_PATH -not -path $ROOT_PATH'/tests/ci*' -not -path $ROOT_PATH'/docker*' -not -path $ROOT_PATH'/contrib*' -name Dockerfile -type f 2>/dev/null | xargs --no-run-if-empty -n1 echo "Please move Dockerfile to docker directory:"
```
Comment (Felixoid) [inline utils/check-style/check-style:289]: Let's add here another directory instead of removing the check.

File: utils/check-style/check_cpp.sh [hunk 2]
```diff
@@ -22,10 +22,11 @@ start=$(date +%s)
 runtime=$(($(date +%s)-start))
 echo "Check whitespaces. Done. $runtime seconds."
 
-start=$(date +%s)
-./check-workflows             |& tee /test_output/workflows_output.txt
-runtime=$(($(date +%s)-start))
-echo "Check workflows. Done. $runtime seconds."
+# Disabled for praktika integration. To be removed or re-enabled
```
Comment (Felixoid) [inline utils/check-style/check_cpp.sh:25]: Unless we're moved completely, let's keep it checked.

**Review by Felixoid — COMMENTED** [review]
There are two destructive changes, let's not break existing style check yet

File: utils/check-style/check-style (hunk 1 above)
Comment (maxknv) [reply utils/check-style/check-style:289]: author is fine with removal

**Pull request merged**